### ISSUE TITLE
Footer when reviewed on Mobile. 

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1657]
**Issue**:
You can open and close the menu by swiping right and then hit the "X" to close the menu but all it does is keep opening.

Not a failure but the social media icons do not line up, may just be a spacing issue. (Mobile iphone SE)

**Comment by maya** [comment]
There's no menu on the footer, nor an X to close it.

> Not a failure but the social media icons do not line up, may just be a spacing issue. (Mobile iphone SE)

They line up fine for me:
<img width="296" alt="screen shot 2018-02-12 at 3 50 59 pm" src="https://user-images.githubusercontent.com/5249443/36126243-8a00bbae-100c-11e8-976e-3626cef5f139.png">

Feel free to provide more details, but will close it for now as I cannot reproduce.
